var/spack/repos/builtin/packages/cntk/package.py

```diff
@@ -49,6 +54,26 @@ class Cntk(Package):
     # Patch to change behaviour of lock file - https://github.com/Microsoft/CNTK/issues/62
     patch('lock-file.patch')
 
+    # It seems that cntk, at least version 2.0, can not be built with GCC
+    # beyond 4.8.5.
+    conflicts('%gcc@5:')
+
+    def patch(self):
+        if 'protobuf+shared' in self.spec:
+            library_suffix = 'so'
```

Comment:
Is this also true on macOS?

Reply:
This will not be necessary at all with other change s being made.